src/main/java/io/anserini/document/JsonDocument.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Anserini: An information retrieval toolkit built on Lucene
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.document;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+/**
+ *
+ * Json adaptor
+ * Inside each file is either a JSON Object (one document) or a JSON Array (multiple documents)
+ * Example of JSON Object:
+ * {
+ *   "id": "doc1",
+ *   "contents": "this is the contents."
+ * }
+ * Example of JSON Array:
+ * [
```

Comment:
Will we accept the convention of one JSON per line, but not wrapped inside an array?
This is a pretty common case also - e.g., what comes out of the Twitter streaming API.